beacon-chain/forkchoice/doubly-linked-tree/epbs_test.go

```diff
@@ -79,6 +81,117 @@ func TestStore_Insert_PayloadContent(t *testing.T) {
 	require.Equal(t, n, n2)
 }
 
+func TestStore_Insert_PayloadEnvelope(t *testing.T) {
+	ctx := context.Background()
+	f := setup(0, 0)
+	s := f.store
+	// The tree root is full
+	fr := [32]byte{}
+	n := s.nodeByRoot[fr]
+	require.Equal(t, true, n.isParentFull())
+
+	// Insert a child
+	cr := [32]byte{'a'}
+	cp := [32]byte{'p'}
+	n, err := s.insert(ctx, 1, cr, fr, [32]byte{}, fr, 0, 0)
```

Comment:
I'd like to suggest to rename insert's parameter name `parentHash` to `parentPayloadHash`. There are different kinds of hash like BlockHash.

Reply:
blockhash and payloadhash are the same, hash is for the EL, root for the CL